bindings/assembly/nearEntry.ts

```diff
@@ -0,0 +1,268 @@
+
+import { storage, near, base64, base58, logging } from 'near-runtime-ts';
+import { JSONEncoder } from "assemblyscript-json";
+import { JSONDecoder, ThrowingJSONHandler, DecoderState } from "assemblyscript-json";
+// Runtime functions
+// tslint:disable: no-unsafe-any
+
+
+
+type Usize = u64;
```

Comment:
Why do we need this?

Reply:
Probably not, but before I had to switch before when testing in non `u32` environments and thought this might be useful in the future.